x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/parser/LogicalPlanBuilder.java

```diff
@@ -243,4 +258,8 @@ public LogicalPlan visitTableName(TableNameContext ctx) {
         TableIdentifier tableIdentifier = visitTableIdentifier(ctx.tableIdentifier());
         return new UnresolvedRelation(source(ctx), tableIdentifier, alias, ctx.FROZEN() != null);
     }
+
+    private Limit wrapWithLimit(LogicalPlan plan, Source source, Token limit) {
```

Comment:
I'd prefer calling the method simply `limit` - it's not really wrapping but instead adding a new node to the tree.